### PR TITLE
Clean up FileAPI idlharness tests

### PR DESCRIPTION
**Pull request by lukebjerring**
After fixing https://github.com/web-platform-tests/wpt/issues/10338, we can leverage `add_dependency_idls` to import the 'real' untested idl from sources cleanly.

### CODE REVIEW

**Comment by lukebjerring** [comment]
Note, also, that #11419 caused a test failure to show up for FileAPI (since the untested original interface didn't have an `Exposed` attr). 

> FAIL Partial interface URL: valid exposure set Partial URL interface is exposed to 'DedicatedWorker', the original interface is not.

https://test-results.appspot.com/data/layout_results/linux_chromium_rel_ng/113144/layout-test-results/results.html

This change fixes that.

File: FileAPI/idlharness.html
Comment (inexorabletash) [inline FileAPI/idlharness.html]: Doesn't this still need `.files` ?

**Comment by lukebjerring** [comment]
Chromium CI equivalent (except the manual test):
https://chromium-review.googlesource.com/c/chromium/src/+/1094416

**Review by foolip — APPROVED** [review]
Fixed indentation, LGTM, not sure if @inexorabletash wants a second look?

**Review by inexorabletash — APPROVED** [review]
lgtm; the linked chromium CI's worker test times out; did that get fixed?

**Comment by lukebjerring** [comment]
Re-running now, but not particularly worried since the other test passed fine. Feel free to wait on re-run, or merge if happy as-is.

**Pull request merged**
